bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/TestClusterAPI.java

```diff
@@ -147,17 +148,33 @@ private Cluster getCluster(BigtableClusterAdminClient client, String clusterName
     }
   }
 
+  // This is needed to see if there is an error.  The *_NUMBER fields are 1 based, but the
+  // getFields() list is 0 based, so we need to subtract 1 from Operation.ERROR_FIELD_NUMBER 
+  // to get the correct ERROR_DESCRIPTOR
+  final static FieldDescriptor ERROR_DESCRIPTOR =
+      Operation.getDescriptor().getFields().get(Operation.ERROR_FIELD_NUMBER - 1);
+
   private void waitForOperation(BigtableClusterAdminClient client, String operationName,
-      int maxSeconds) {
+      int maxSeconds) throws InterruptedException {
     GetOperationRequest request = GetOperationRequest.newBuilder().setName(operationName).build();
     for (int i = 0; i < maxSeconds; i++) {
       try {
         Thread.sleep(1000);
       } catch (InterruptedException e) {
-        e.printStackTrace();
+        Thread.interrupted();
```

Comment:
why is this here? Now that the method throws Interrupted, you don't need the try statement.